process/rewardTransaction/interceptedRewardTransaction.go

```diff
@@ -96,16 +96,9 @@ func (inRTx *InterceptedRewardTransaction) processFields(rewardTxBuff []byte) er
 
 // integrity checks for not nil fields and negative value
 func (inRTx *InterceptedRewardTransaction) integrity() error {
```

Comment:
This integrity function is no longer needed. Might call directly inRTx.rTx.CheckIntegrity() where integrity function was used

Reply:
done